src/renderer/pages/UploaderConfigPage.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <div id="config-list-view">
+    <div class="view-title">
+      {{ $T('SETTINGS') }}
+    </div>
+    <el-row :gutter="15" justify="space-between" align="center" type="flex" class="config-list">
+      <el-col
+        class="config-item-col"
+        v-for="item in curConfigList"
+        :key="item._id"
+        :span="11"
+        :offset="1"
+      >
+        <div
+          :class="`config-item ${defaultConfigId === item._id ? 'selected' : ''}`"
+          @click="() => selectItem(item._id)"
+        >
+          <div class="config-name">{{item._configName}}</div>
+          <div class="config-update-time">{{formatTime(item._updatedAt)}}</div>
+          <div v-if="defaultConfigId === item._id" class="default-text">{{$T('SELECTED_SETTING_HINT')}}</div>
+          <div class="operation-container">
+            <i class="el-icon-edit" @click="openEditPage(item._id)"></i>
+            <i :class="`el-icon-delete ${curConfigList.length <= 1 ? 'disabled' : ''}`" @click.stop="() => deleteConfig(item._id)"></i>
+          </div>
+        </div>
+      </el-col>
+      <el-col
+        class="config-item-col"
+        :span="11"
+        :offset="1"
+      >
+        <div
+          class="config-item config-item-add"
+          @click="addNewConfig"
+        >
+          <i class="el-icon-plus"></i>
+        </div>
+      </el-col>
+    </el-row>
+    <el-row type="flex" justify="center" :span="24" class="set-default-container">
+      <el-button class="set-default-btn" type="success" @click="setDefaultPicBed(type)" round :disabled="defaultPicBed === type">{{ $T('SETTINGS_SET_DEFAULT_PICBED') }}</el-button>
+    </el-row>
+  </div>
+</template>
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import dayjs from 'dayjs'
+import { completeUploaderMetaConfig } from '../utils/uploader'
+
+@Component({
+  name: 'OssConfigList'
```

Comment:
change this name

Reply:
& should add mixin for `defaultPicBed` just like `picbeds/index.vue` do

Reply:
![image](https://user-images.githubusercontent.com/12621342/203200024-2a8f5c4c-e044-4867-a909-1fa20fdbef73.png)

or will cause this error:

![image](https://user-images.githubusercontent.com/12621342/203200053-ab604411-5727-4c09-8b6c-fd0db8638027.png)